test/fixtures/automl.py

```diff
@@ -43,8 +44,8 @@ def _create_automl(
         "time_left_for_this_task": 30,
         "per_run_time_limit": 5,
         "seed": DEFAULT_SEED,
-        "n_jobs": 2,
-        "ensemble_size": 10,
+        "ensemble_class": EnsembleSelection,
+        "ensemble_kwargs": {"ensemble_size": 10},
```

Comment:
I hope this wasn't too difficult to figure out during the change. Thankfully it means it only needs to change in one place for all the tests for the `AutoML` class

Reply:
Yes, it's great that there's only one place to change this. No, it wasn't hard to find out, I only had to search for occurences of `ensemble_size=10`.